docs/GettingStarted.md

```diff
@@ -62,6 +61,8 @@ $ java -jar generator.jar --help
 
 or click [here](UserGuide.md#command-line-arguments) to find the full range of command line arguments.
 
+Alternatively you can use the [datahelix playground](https://finos.github.io/datahelix/playground/) to get a feel for what it is like to run the profile without downloading the JAR. Please note that the playground does not support all features and is not intended to be a substitute for downloading the JAR.
```

Comment:
I'm not sure about the wording here because it might make people think the playground does not support all *profile* features, which it does.  Better to be more explicit and say something like "the playground does not support changing your command-line options"

Reply:
I don't think the faker supports features which require files to use. I couldn't figure out how to get it to work with an inMap constraint for example.

Reply:
Good point, it won't work with that - I think that's the only case.  Maybe it needs rewording something like "Although the playground supports almost all features, there are some it does not, and we don't intend it to be a substitute for downloading and running the Datahelix Generator yourself."

Reply:
> I couldn't figure out how to get it to work with an inMap constraint for example

Reply:
> > I couldn't figure out how to get it to work with an inMap constraint for example
> 
> On that note - do we have an open issue around features that require multiple files? I'd like to have an in-line equivalent to all of those (post v1 release!)

I don't think we do have a ticket for specifically introducing in-line equivalents - I assume you mean any feature that requires another file to be referenced from a profile file?

Reply:
I've reworded the sentence as you suggested @willsalt-sl